server/controllers/notificationController.js

```diff
@@ -0,0 +1,135 @@
+import models from '../models';
+
+const { Notifications } = models;
+
+const notificationController = {
+  /**
+  * @description notification controller
+  * @param  {object} req The HTTP request object
+  * @param  {object} res The HTTP response object
+  * @returns {object} The HTTP response object
+  */
+  getUnreadAll: (req, res) => {
+    Notifications
+      .findAll({
+        where: {
+          receiverId: req.currentUser.id
+        }
+      })
+      .then((notifications) => {
+        const count = notifications.length;
+        return res.status(200).jsend.success({
+          count,
+          notifications
+        });
+      })
+      .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the serve'));
+  },
+  getAll: (req, res) => {
+    Notifications.scope(null)
+      .findAll({
+        where: {
+          receiverId: req.currentUser.id
+        }
+      })
+      .then((notifications) => {
+        const count = notifications.length;
+        return res.status(200).jsend.success({
+          count,
+          notifications
+        });
+      })
+      .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the serve'));
+  },
+  getOne: (req, res) => {
+    Notifications.scope(null)
+      .findOne({
+        where: {
+          id: req.params.notifyId,
+          receiverId: req.currentUser.id
+        }
+      })
+      .then((notify) => {
+        if (!notify) {
+          return res.redirect('/api/v1/notifications');
+        }
+        notify.isRead = true;
+        notify.save()
+          .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the serve'));
```

Comment:
Seems like they're a couple of the same wrong spellings here. Please do look into it.

Reply:
I will make the necessary changes